src/OmniSharp.Abstractions/Models/v1/TypeLookup/DocumentationComment.cs

```diff
@@ -10,15 +10,15 @@ namespace OmniSharp.Models.TypeLookup
     public class DocumentationComment
     {
         public string SummaryText { get; }
-        public string[] TypeParamElements { get; }
-        public string[] ParamElements { get; }
+        public DocumentedObject[] TypeParamElements { get; }
```

Comment:
This could just be Array<KeyValuePair<string, string>> or Array<Tuple<string, string>>

Reply:
Thoughts @DustinCampbell ?

Reply:
I'm not sure what Newtownsoft.Json supports here. If we go with a custom type, I might call it `DocumentationItem`.

Reply:
@akshita31 : Note that Array<T> is not a thing in C#. @rchande meant `KeyValuePair<string, string>>[]` and `Tuple<string, string>[]`. I would lean away from tuple because it would mean that the protocol would have weird names like `Item1` and `Item2` in it, which isn't what we want.